evap/results/views.py

```diff
@@ -142,7 +142,7 @@ def course_detail(request, semester_id, course_id):
 def user_can_see_text_answer(user, represented_users, text_answer, public_view=False):
     if public_view:
         return False
-    if text_answer.state not in COMMENT_STATES_REQUIRED_FOR_VISIBILITY:
+    if text_answer.state not in [TextAnswer.PRIVATE, TextAnswer.PUBLISHED]:
```

Comment:
while this is correct, i think it's weird to let calculate_results hand out comments that are not supposed to be shown anywhere. how about we put the filtering in there and here use an assert instead at the beginning of the method? i would continue explicitly listing the allowed states and not use a variable so the assert must manually be updated if need be.